dask_cloudprovider/providers/aws/ecs.py

```diff
@@ -366,7 +366,7 @@ def __init__(self, scheduler: str, cpu: int, mem: int, gpu: int, **kwargs):
                 "--name",
                 str(self.name),
                 "--nthreads",
-                "{}".format(int(self._cpu / 1024)),
+                "{}".format(int(self._cpu / 1024) if self._cpu >= 1024 else 1),
```

Comment:
Thanks @gvelchuru! One small comment: could you switch this to something like:

```python
min(int(self._cpu / 1024), 1)
```

which is more concise